### DISCUSSION TITLE
Issues Receiving Sensor Signals with RTL-SDR and RTL_433 on Debian

### DISCUSSION THREAD

**Discussion by Think-Wave** [merbanan/rtl_433#2851] [Q&A]
Hello everyone,

I'm attempting to receive data from my sensors using an RTL-SDR Blog brand R820T2 RTL2832U 1PPM TCXO SMA V4 Dongle on a Debian server (virtual machine on Unraid). Initially, I tried this on a VM for Home Assistant without success. Connection and data reception through SDR# and the RTL_433 plugin on a Windows system work perfectly, confirming that the hardware is in order.

Setup:

Dongle: RTL-SDR Blog brand R828D RTL2832U 1PPM TCXO SMA V4
Server OS: Debian (VM on Unraid)
Software: librtlsdr 0.8, rtl_433
After installing RTL_SDR and rtl_433 on Debian, I start rtl_433 in the console. There are no error messages, but no sensor signals are being decoded either.

Does anyone have experience with this setup or any idea what could be the issue? Any hints or suggestions would be greatly appreciated!

Thanks in advance!

**Comment by zuckschwerdt** [comment]
To get more info use `rtl_433 -v -Y autolevel -M level -M noise`.
Replace with `-vv` or `-vvv` to see what the decoders are doing, as last resort use `-A` to really show everything received.

**Comment by Think-Wave** [comment]
Hello, thank you for your prompt response. I followed your advice and received this output.
![Screenshot 2024-02-25 150614](https://github.com/merbanan/rtl_433/assets/35407155/87a5ad18-e5f5-4f7f-94bc-c50d6b0bf36c)

**Reply by zuckschwerdt** [reply]
That's a massively jammed band. You want at least -15, better -20 dB.

**Reply by Think-Wave** [reply]
Same dongle, same position of the antenna, same antenna, just a different PC which is my Windows PC, otherwise everything is exactly the same.
![Screenshot 2024-02-24 200926](https://github.com/merbanan/rtl_433/assets/35407155/c8dd4510-b5fb-45f6-a7eb-fbb88f5688be)

**Reply by zuckschwerdt** [reply]
Did you install a Blog-V4 enabled librtlsdr?

**Reply by Think-Wave** [reply]
I have used these here.

https://github.com/librtlsdr/librtlsdr

**Reply by zuckschwerdt** [reply]
That's the enhanced version and has Blog-V4 support. Very strange. Record a short sample and view on https://triq.org/pdv/
https://triq.org/rtl_433/ANALYZE.html#grab-a-sample

**Reply by Think-Wave** [reply]
I managed to get it working with another SDR stick, only needed an adapter for the antenna which arrived today, and it ran without any issues. I've already integrated it into my Home Assistant. Unfortunately, I didn't know that the air pressure sensor is in the main station. I did the test and had the dB values displayed, and they were between -15 and -20 dB :) The manual for the V4 dongle mentioned that they do not run on every system yet.

![image](https://github.com/merbanan/rtl_433/assets/35407155/02e90bf4-0179-4d83-877b-37ddcda265e9)

Oh, regarding the strange measurements, that was just a test; I need a new sensor as this one has been damaged by a careless person. I believe the temperature sensor is the only one showing accurate values :D


thx for help :) i am happy <3

**Reply by zuckschwerdt** [reply]
Happy to hear that the problem was found. Maybe better support will be added to the rtlsdr libs at some point.

**Reply by Think-Wave** [reply]
Yes, that would be a positive development :)
